components/eam/src/dynamics/se/inidat.F90

```diff
@@ -246,16 +252,17 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
     end do
 
     fieldname = 'T'
-    tmp_point = 0.0_r8
+    tmp = 0.0_r8
 
+    call t_startf('read_inidat_infld')
     if (.not. scm_multcols) then
       call infld(fieldname, ncid_ini, ncol_name, 'lev', 1, npsq,          &
-           1, nlev, 1, nelemd, tmp, found, gridname='GLL')
+           1, nlev, 1, nelemd, tmp, found, gridname=grid_name)
     else
-      tmp_point = 0.0_r8
       call infld(fieldname, ncid_ini, ncol_name, 'lev', 1, 1,          &
```

Comment:
I'm confused why this block of code was modified.  This will cause DP-SCREAM initialization to fail (just verified with a test run).  Unless I'm missing something...